src/EditorFeatures/CSharpTest/Diagnostics/GenerateVariable/GenerateVariableTests.cs

```diff
@@ -1365,6 +1365,27 @@ void M()
 @"class Foo { public int Gibberish { get ; internal set ; } } class Bar { void foo ( ) { var c = new Foo { Gibberish = 24 } ; } } ");
         }
 
+        [WorkItem(13166, "https://github.com/dotnet/roslyn/issues/13166")]
```

Comment:
VB does not have nested object initializers.  And it has tests for normal object initializers.